frontend/consumer-ui/components/Map.stories.js

```diff
@@ -0,0 +1,14 @@
+import { storiesOf } from '@storybook/vue'
+import Map from './Map'
+
+const components = {
+  'shops-map': Map
+}
+
+storiesOf('Map', module)
+  .add('map with shop markers', () => {
+    return {
+      components,
+      template: '<shops-map/>'
+    }
+  })
```

Comment:
That's our first story file. It displays the `Map` component in a frame and makes it available for visualisation, manual testing, snapshot testing, etc.